test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-base_global_discount @ git+https://github.com/OCA/server-backend.git@refs/pull/221/head#subdirectory=setup/base_global_discount
```

Comment:
This module is already merged, could you please remove this file in order to pass the tests in Github Actions.